Backend/components/graph_api.py

```diff
@@ -6,9 +6,15 @@
 @router.get("/graph")
 async def get_graph(data: str, start_time: int, end_time: int):
     """pass in a comma separated list of data that the frontend requests start_time end_time must be in unix millisec"""
+    if not data:
+        # use a query to get timestamp
+        query_result = await db.query(['speed'], start_time, end_time, ['avg'],
+                                      (end_time - start_time) // 60 or 1)
+        return {'response': {"timestamps": query_result.index.to_list()}}
     data = [d.strip() for d in data.split(',')]
-    query_result = await db.query(data, start_time, end_time, ['avg']*len(data), (end_time-start_time)//5000 or 1)
-    result = {"timestamp": query_result.index.to_list()}
+    query_result = await db.query(data, start_time, end_time, ['avg']*len(data), (end_time-start_time)//60 or 1)
+    result = {"timestamps": query_result.index.to_list()}
```

Comment:
We need to use min-max decimation for booleans, and we should ensure that any outstanding/non-nominal values are shown on the graph. For reference, this is a boolean value:
![image](https://github.com/badgerloop-software/chase-car-dashboard/assets/72111610/294c1c3f-f7a1-4bb4-b0ba-95d81316d013)

Reply:
Yeah, I'm thinking adding a new field to dataformat to specify the aggregation algorithm for the data

Reply:
Good idea

Reply:
@MCLiii, once this is added (or at least when we can use a min-max algorithm for booleans and an average for other values), the PR should pretty much be ready to go

Reply:
Boolean aggregation is fixed